server/app.js

```diff
@@ -273,6 +280,11 @@ app.get("/trees", async function (req, res) {
                   SELECT region_id, zoom_level
                   FROM active_tree_region
                   WHERE zoom_level = $1
+                  ${treeIds && treeIds.length > 0 ?
```

Comment:
Here is about finding the next zoom target region center, I forgot the filter the region by trees that belongs to the organization.